src/enums/furnished.ts

```diff
@@ -0,0 +1,5 @@
+/** Enum for Furnished codes. */
+export enum Furnished {
+  N = 'N',
+  Y = 'Y'
+}
```

Comment:
Simple, yet, but it forces other code to use this type appropriately.

Reply:
Is N and Y representative of 'no' and 'yes'?
Could we elaborate on what Furnished is a little bit in the comment maybe? :)

Reply:
> Is N and Y representative of 'no' and 'yes'?

Y.

> Could we elaborate on what Furnished is a little bit in the comment maybe? :)

I will ask Linda and update the comment.

Reply:
Awesome :) Thanks

Reply:
Updated!